network/bridge/bridge.go

```diff
@@ -0,0 +1,57 @@
+package bridge
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/daemon/mgr"
+	"github.com/alibaba/pouch/network"
+
+	"github.com/docker/libnetwork/drivers/bridge"
+	"github.com/docker/libnetwork/netlabel"
+)
+
+// New is used to initiaze bridge network.
+func New(ctx context.Context, config network.BridgeConfig, manager mgr.NetworkMgr) error {
+	// TODO: use config to set bridge network.
+	// TODO: support ipv6.
+
+	// clear exist bridge network
+	if err := manager.NetworkRemove(ctx, "bridge"); err != nil {
+		return err
+	}
+
+	ipamV4Conf := &types.IPAMConfig{
+		AuxAddress: map[string]string{},
+		Subnet:     DefaultSubnet,
+		IPRange:    DefaultIPRange,
+		Gateway:    DefaultGateway,
+	}
+
+	ipam := &types.IPAM{
+		Driver: "default",
+		Config: []*types.IPAMConfig{ipamV4Conf},
+	}
+
+	networkCreate := &types.NetworkCreate{
+		Driver:     "bridge",
+		EnableIPV6: false,
+		Options: map[string]string{
+			bridge.BridgeName:       DefaultBridge,
+			bridge.DefaultBridge:    strconv.FormatBool(true),
```

Comment:
@rudyfly 
Pouchd can not initialize successfully with the following error:

`failed to create network: bridge device with non default name p0 must be created manually`

and I check the code of libnetwork find that:
````go
	if config.BridgeName != DefaultBridgeName && config.DefaultBridge {
		return NonDefaultBridgeExistError(config.BridgeName)
	}
````
that means if the name of bridge network is not "docker0" and the DefaultBridge field of config is true,  the creation will fail.

So bridge.DefaultBridge should be `false`

Reply:
we should use brctl to create p0 bridge manually, next I will use netlink to create the 'p0' bridge when bridge is not exist.